src/dmd/backend/inliner.d

```diff
@@ -80,6 +79,11 @@ bool canInlineFunction(Symbol *sfunc)
             return false;
         if (config.ehmethod == EHmethod.EH_WIN32 && !(f.Fflags3 & Feh_none))
             return false;       // not working properly, so don't inline it
+
+        static if (0) // disable for the moment
```

Comment:
Changes on `canInlineExpression` are not covered, perhaps this should be removed.

Reply:
I want to re-enable it on the next PR.